app/src/main/java/org/oppia/app/topic/FractionParsingErrors.kt

```diff
@@ -0,0 +1,23 @@
+package org.oppia.app.topic
+
+import org.oppia.app.customview.interaction.FractionInputInteractionView
+
+/** Enum to store the errors of [FractionInputInteractionView]. */
+enum class FractionParsingErrors(errorCode: Int, error: String) {
+  VALID(errorCode = 200, error = "Valid"),
+  INVALID_CHARS(errorCode = 100, error = "Please only use numerical digits, spaces or forward slashes (/)"),
```

Comment:
We should use a string resource for these errors rather than hardcoded strings to ensure that they can be translated.

Reply:
Done